tests/config/test_sets.py

```diff
@@ -150,3 +156,8 @@ def __init__(self, conf: Config):
     assert repr(conf) == "MagicConfigSet(loaders=[IniLoader(section=<Section: magic>, overrides={})])"
 
     assert isinstance(result.state.conf.options, Parsed)
+
+
+def test_do_not_allow_create_config_set(mocker: MockerFixture) -> None:
+    with pytest.raises(TypeError, match="Can't instantiate"):
+        ConfigSet(mocker.create_autospec(Config))  # type: ignore # the type checker also warns that ABC
```

Comment:
Looks like the comment is cut off... `that ABC`... does what?

Reply:
It's short but I thought it's enough, so it's on purpose.